### PR TITLE
Install timedatex on rhel-8-4 image

### PR DESCRIPTION
**Pull request by marusak**
It stopped being included by default and NTP is not working without it.
See https://bugzilla.redhat.com/show_bug.cgi?id=1918340

 * [x] image-refresh rhel-8-4

### CODE REVIEW

**Comment by cockpituous** [comment]
image-refresh in progress on 2-ci-srv-06.
Log: https://logs.cockpit-project.org/logs/image-refresh-1572-20210122-092626/

**Comment by cockpituous** [comment]
image-refresh rhel-8-4 done: https://github.com/cockpituous/bots/commits/image-refresh-rhel-8-4-20210122-095252

File: images/scripts/rhel.setup [hunk 1]
```diff
@@ -306,6 +306,12 @@ else
     dnf --setopt=install_weak_deps=False install -y cockpit
 fi
 
+# HACK: 8.4 has broken NTP
+# https://bugzilla.redhat.com/show_bug.cgi?id=1918340
+if [ "$IMAGE" = "rhel-8-4" ]; then
+    dnf install -y timedatex
+fi
```
Comment (martinpitt) [inline images/scripts/rhel.setup:313]: Hmm, I'd just have added it to `TEST_PACKAGES`. It's not a hack around an OS deficiency, it's just an assumption of cockpit's tests IMHO. (fine for me to not rebuild the image for that, as the end result is the same)

**Review by martinpitt — CHANGES_REQUESTED** [review]

File: images/scripts/rhel.setup (hunk 1 above)
Comment (martinpitt) [reply images/scripts/rhel.setup:313]: Oh, nevermind, I should have read the bug first. Sorry!

**Review by martinpitt — APPROVED** [review]
(Assuming tests go green, of course)

**Comment by marusak** [comment]
[testVdo](https://logs.cockpit-project.org/logs/pull-1572-20210122-095849-34f0f6ca-rhel-8-4-cockpit-project-cockpit/log.html#284-2) which is one of out top flakes.

**Pull request merged**